Extend AAS Linux Wrapper Retirement Date (#32280)
DataDog/documentation: content/en/serverless/guide/azure_app_service_linux_code_wrapper_script.md

<div class="alert alert-danger">
The AAS Linux Wrapper is now deprecated. It will continue to receive layer bumps but no new features.
It will be retired on March 1, 2026, at which point no further updates will be provided.

> You can continue to use the wrapper after March 1, 2026. However, no further updates, fixes, or support will be provided beyond that point.

Reviewer: ```suggestion
You can continue to use the wrapper after March 1, 2026, but its use will not be supported.
```
